tutorial/readme.rst

```diff
@@ -23,3 +23,4 @@ and IPython (v1.0+). Note that only `yt-3.0 <https://bitbucket.org/yt_analysis/y
 supports PyNE.  Be sure to use this version of yt rather 
 than the more stable v2.x series.
 
+"""I suggest adding a reference for people to contact incase they have questions here: _________"""
```

Comment:
Rather than have then contact people individually, I'd actually suggest that they submit a ticket in the issues system or send an email to pyne-dev. Perhaps the text you have above would be appropriate here: 

If you have any questions about this process, don't hesitate to ask the mailing list (https://groups.google.com/forum/#!forum/pyne-dev, pyne-dev@googlegroups.com).		  pyne-dev@googlegroups.com). Please provide as clear of an explanation (with examples if possible) of any problems or questions you might have.